rosyk_task5.py

```diff
@@ -0,0 +1,44 @@
+# Multiparadigm programming languages, Task 2
+# Rosyk Nikita, №15
+
+
+def row_sum(eps):
+    series_sum = 0
+    n = 1
+    row_number = 1
+    try:
+        while abs(row_number) >= eps:
+            row_number = n / (n - 1) ** 2
```

Comment:
look, you initially set n as 1. Then in the first iteration you has (n - 1) ** 2, so it should be 0.  I suspect you are going to receive zeroDivision in the first iteration, aren't you?